test/playlist.test.js

```diff
@@ -574,7 +574,7 @@ QUnit.test('safeLiveIndex accounts for liveEdgePadding in simple case', function
   );
 
   assert.equal(
-    Playlist.safeLiveIndex(playlist, 0), 6,
+    Playlist.safeLiveIndex(playlist, 0), 5,
     'returns 6 when liveEdgePadding is 0 and duration is 6'
```

Comment:
Was there a reason for this change? If so, we may have to update the message.

Reply:
so this was actually a bug. We return `playlists.segments.length` which isn't a valid index in the array, it was 1 too high.